pig/data.py

```diff
@@ -212,22 +207,14 @@ def _raw_clips(self):
                     clips = pig.preprocess.segment(video, duration=self.duration, jitter=self.jitter)
                 for clip in clips:
                     yield clip
-                                       
-
-    def _positives(self, items):
```

Comment:
I deleted this function because to my mind it wasn't doing anything currently. Hope it doesn't break anything!